charts/posthog/templates/ingress.yaml

```
@@ -85,6 +85,9 @@ spec:
           - pathType: Prefix
             path: "/e"
```

Maybe do this instead?

```suggestion
            path: "/e/"
```

Otherwise we have e.g. /events/ going to posthog-events as well which isn't correct.

---

(although I guess we need to make sure that you need the trailing slash for that endpoint 🤔 )

---

I agree but that's fundamentally another routing issue 😁 

Should we change all the non CLB with

```
diff --git a/charts/posthog/templates/ingress.yaml b/charts/posthog/templates/ingress.yaml
index ca04d8b..ad06f99 100644
--- a/charts/posthog/templates/ingress.yaml
+++ b/charts/posthog/templates/ingress.yaml
@@ -77,22 +77,22 @@ spec:
                   number: {{ .Values.service.externalPort }}
           {{- if (ne (include "ingress.type" .) "clb") }}
           - pathType: Prefix
-            path: "/capture"
+            path: "/capture/"
             backend: *INGESTION
           - pathType: Prefix
-            path: "/decide"
+            path: "/decide/"
             backend: *INGESTION
           - pathType: Prefix
-            path: "/e"
+            path: "/e/"
             backend: *INGESTION
           - pathType: Prefix
-            path: "/engage"
+            path: "/engage/"
             backend: *INGESTION
           - pathType: Prefix
-            path: "/track"
+            path: "/track/"
             backend: *INGESTION
           - pathType: Prefix
-            path: "/s"
+            path: "/s/"
             backend: *INGESTION
           {{- else }}
           - pathType: ImplementationSpecific
```

?

---

I'm inclined to merge this as it is and re-evaluate the issue with the other endpoints. Fair point tho!